packages/assets/gulpfile.js

```diff
@@ -53,10 +64,104 @@ function compileMiniScss() {
   .pipe(dest('./css'));
 };
 
+function watchScss() {
+  return watch(['./build/scss/*.scss'], {ignoreInitial: false}, parallel(compileMiniScss, compileScss));
+}
+
+function compileHamburgerHeader() {
+  return src([
+    './build/js/header-hamburger-vendor.js',
+    '../patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js',
+    '../patternlab/styleguide/source/assets/js/modules/mobileNav.js'
+  ])
+    .pipe(babel({
+      presets: ['@babel/env']
+    }))
+    .pipe(concat('header-hamburger.js'))
+    .pipe(dest('./js'));
+}
+
+function compileMiniHamburgerHeader() {
+  return src([
+    './build/js/header-hamburger-vendor.js',
+    '../patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js',
+    '../patternlab/styleguide/source/assets/js/modules/mobileNav.js'
+  ])
+    .pipe(babel({
+      presets: ['@babel/env']
+    }))
+    .pipe(concat('header-hamburger.min.js'))
+    .pipe(terser())
+    .pipe(dest('./js'));
+}
+
+function compileMainNav() {
+  return src([
+    '../patternlab/styleguide/source/assets/js/modules/mainNav.js'
+  ])
+  .pipe(babel({
+    plugins: [
+      'babel-plugin-rewire-exports',
```

Comment:
why mainNav.js needs to be transpiled differently?

Reply:
This is because I need to remove the export statement only for this file while preserving the IIFE syntax.